lua-docs/content/reference/ray.yml

```diff
@@ -21,12 +21,37 @@ constructors:
 
 functions:
     - name: "Cast"
+      argument-sets: 
+        -
+        -
+          - name: "filterIn"
+            type: "CollisionGroups"
+        -
+          - name: "filterIn"
+            type: "Shape"
+        -
+          - name: "filterIn"
+            type: "nil"
+          - name: "filterOut"
+            type: "Object"
+        -
+          - name: "filterIn"
+            type: "CollisionGroups"
+          - name: "filterOut"
+            type: "Object"
+        -
+          - name: "filterIn"
+            type: "Shape"
+          - name: "filterOut"
+            type: "Object"
```

Comment:
Yes, ideally I would like to indicate different possible types for each parameters. 
This will be an improvement for the docs generator. Let's merge as is in the meantime.